src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/models/CCJContent.java

```diff
@@ -16,8 +16,8 @@
     private String requestedAt;
     private String requestedDate;
     private AmountContent amount;
-    private String repaymentOption;
     private String defendantDateOfBirth;
+    private RepaymentPlanContent repaymentPlanContent;
```

Comment:
We do not tend to suffix all fields with content. To stick to that convention it should have been named `repaymentPlan`.

REC